docs/releases.rst

```diff
@@ -4,7 +4,57 @@
     Releases
 ======================
 
-fmf-1.4
+
+fmf-1.5.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``+`` operator now can be used for merging ``list`` of
+dictionaries with a single ``dict``. This can be for example
+useful when extending the ``discover`` step config which defines
+several phases:
+
+.. code-block:: yaml
+
+    discover:
+      - how: fmf
+        url: https://github.com/project/one
+      - how: fmf
+        url: https://github.com/project/two
+
+    /tier1:
+        summary: Run tier one tests
+        discover+:
+            filter: "tier:1"
+
+    /tier2:
+        summary: Run tier two tests
+        discover+:
+            filter: "tier:2"
+
+See the :ref:`merging<merging>` section for more details and
+examples.
+
+The ``-`` operator no more raises exception when the parent node
+does not define the key. This allows reducing values even for
+cases when user does not have write permissions for the parent
+data. For example, in order to make sure that the ``mysql``
+package is not included in the list of required or recommended
+packages you can now safely use this:
+
+.. code-block:: yaml
+
+    discover:
+        how: fmf
+        adjust-tests:
+          - require-: [mysql]
```

Comment:
Do we have some "standard" for single item arrays in documentation? It would be good to have the same style everywhere (not strictly related to this PR)
Reminded of https://github.com/teemtee/tmt/pull/2187 :)

Reply:
I don't think we have any standard. And for short lists I'd say this notation is more readable/concise.